app/concepts/todo_list/create.rb

```diff
@@ -0,0 +1,10 @@
+class TodoList::Create < Trailblazer::Operation
+  include Model
+  model TodoList
+
+  def process(params)
```

Comment:
I like the idea on how you decided to create the `User` here, but note that it would be better if this operation only speaks for itself - you could just pass the created user before doing this operation since the models are related to one another, and that's how associations should work. Also, I noticed that you assign something for `@model`, but that's not needed because it was already declared via `model TodoList` from above.